### PR TITLE
Add temporary fix for broken Drupal

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR revises the build so that the build continues even when Drupal Dev is down or returning unexpected data. In this case, Drupal is returning `null` values as pages. In the future, the build should definitely fail when this happens, but this is needlessly breaking things at the moment.

## Screenshots
image

## Acceptance criteria
- [ ] Build works again

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]

**Comment by ncksllvn** [comment]
The fix for this PR was blocked by an error fixed by https://github.com/department-of-veterans-affairs/vets-website/pull/9373, and vice-versa. So, the latter has been merged into this PR so that both fixes can be merged at once.

**Comment by markgreenburg** [comment]
Blocked by https://github.com/department-of-veterans-affairs/vets-website/pull/9372 and vice-versa; Nick merging this branch into his PR and we'll go from there.

**Pull request merged**
